src/components/PWDKChip.tsx

```diff
@@ -0,0 +1,52 @@
+import { Chip } from '@mui/material'
+import { styled } from '@mui/material/styles'
+import { ComponentProps } from 'react'
+
+interface ChipProps extends ComponentProps<typeof Chip> {
+  label: string
+  selectable?: boolean
+  selected?: boolean
+}
```

Comment:
```suggestion
interface ChipProps extends Chip {
  label: string
  selectable?: boolean
  selected?: boolean
}
```
bukannya gini ya harusnya? mui harusnya sih nyediain types nya chip yang kita bisa extend sesuai kebutuhan

Reply:
gabisa a, errornya karena kalo Chip aja dia refers to value, sementara butuhnya type